frontend/docs/application_config.md

```diff
@@ -211,6 +249,62 @@ Introducing dashboards into Amundsen allows users to discovery data analysis tha
 
 After ingesting dashboard metadata into the search and metadata services, set `IndexDashboardsConfig.enabled` to `true` on the application configuration to display the UI for the aforementioned features.
 
+### Index Features
+*TODO
```

Comment:
is there something we need to add here?

Reply:
Yes, there is info for the other index options, but not for this one.
Ideally a short explanation and an example of how to set the flag (trivial, I know)